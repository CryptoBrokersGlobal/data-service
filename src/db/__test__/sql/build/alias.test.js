const sql = require('../../../sql/build/aliases');

describe('Alias sql query build', () => {
  it('`one` should build select by alias', () => {
    expect(sql.one('SOME_ALIAS')).toMatchSnapshot();
  });

  it('`many` should build select by address with order by time', () => {
    expect(sql.many({ address: 'SOME_ADDRESS' })).toMatchSnapshot();
  });
});
