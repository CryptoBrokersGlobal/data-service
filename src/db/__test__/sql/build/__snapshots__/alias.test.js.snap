// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alias sql query build \`many\` should build select by address with order by time 1`] = `"select \\"sender\\" as \\"address\\", \\"alias\\" as \\"alias\\" from \\"txs_11\\" where \\"sender\\" = 'SOME_ADDRESS' order by \\"timeStamp\\" asc limit 1000"`;

exports[`Alias sql query build \`one\` should build select by alias 1`] = `"select \\"sender\\" as \\"address\\", \\"alias\\" as \\"alias\\" from \\"txs_11\\" where \\"alias\\" = 'SOME_ALIAS' limit 1000"`;
